### ISSUE TITLE
[150$] Update project filters

### ISSUE THREAD

**Issue by vic-topcoder** [appirio-tech/connect-app#1374]
1. Update the design of the project filters to match the new UI (darker background.
2. Replace the current drop-down with react-select which allows for multiple selection (see design).
3. Remove the project status (display it as tab); replace it with project segment (we don't have the data yet, but we are going to have a new tag per project - segment - that would be used to filter projects by.

Final design from here:  [https://drive.google.com/drive/folders/1AbIzCycuhTBh7WNCiaD0mYVMXjeGSdd3?usp=sharing](https://drive.google.com/drive/folders/1AbIzCycuhTBh7WNCiaD0mYVMXjeGSdd3?usp=sharing) This is a sketch live design, make sure to download the folder and open the index.html to explore all the artboards.

transition-updates-project-grid-full-list---admin-copy.png (https://topcoder.aha.io/attachments/6493701114636916906/token/1edaf3c0b0a39c59bd0cec742b977ceb2db90c66a5042aaa1a5a55965617f4ac.download?size=original)

Created from Aha! https://topcoder.aha.io/features/TCCONNECT-114

**Comment by aretheregods** [comment]
@gondzo I can finish this, but it's nighttime in New York. Although I could do all of this tonight, it would be best if I finish it tomorrow morning (scheduled events tonight and a need for sleep), which would be outside of the 12 hour window. I neglected accepting the ticket, just to make sure I could actually do it expediently, but I'm not going to have time to deliver quality code unless I do the rest in the morning. So, I'm gonna accept the ticket and just finish in the morning and submit the pull request. I hope it's not unassigned before I get a chance to get back to it.

**Comment by gondzo** [comment]
ok, just keep me updated

**Comment by aretheregods** [comment]
@gondzo I'll get back to working on at 8 a.m. EST, and should be finished not too long after that. As you asked, I'll keep you updated. If I encounter anything prohibitive, I'll just self-unassign.

**Comment by aretheregods** [comment]
@gondzo There are two things of which I'm not sure. First, what options if any should I add to the project segment dropdown? Basically, what are the project segments that should be included in this list? Second, since there's a separate issue for displaying project-statuses as tabs, am I correct in assuming adding that functionality isn't in scope for this issue? Am I just supposed to remove the status dropdown from the filters?

**Comment by aretheregods** [comment]
It's taking me much longer to style the new select menus than I anticipated, so I've reopened this for pickup.

**Comment by maxceem** [comment]
For this task, it would be nice to have a user, which has sufficient credentials. Currently, Parth Manager doesn't have them, so I only can see projects as a regular user http://take.ms/zovGv.
Let me know if can add to the user `Connect Manager` role in admin backend.

**Comment by vikasrohit** [comment]
Earlier `pshah_manager` used to have Manager role. I am not sure what happened but seems like someone has removed his roles. I would try fixing it.

**Comment by maxceem** [comment]
@vikasrohit yes, thank you. I haven't added this role back myself as I was afraid of stepping into somebody working process.

**Comment by vikasrohit** [comment]
@maxceem are you able to add/remove roles for a user?

**Comment by vikasrohit** [comment]
@maxceem I have added back manager role to pshah_manager user.

**Comment by maxceem** [comment]
@vikasrohit yes, I have admin credentials for **development** admin backend, from other challenges. For example [this challenge](https://www.topcoder.com/challenge-details/30060941/?type=develop) provides them.

**Comment by vikasrohit** [comment]
Ohk, got it. It seems like some else having the same credentials changed the roles.

**Comment by maxceem** [comment]
Submitted PR https://github.com/appirio-tech/connect-app/pull/1428

## Implementation notes

- I keep segments list empty so far. I cannot populate it with fake values because API requests will fail as `segment` is not available option for filtering.

- `react-select@0.9.1` is being used in the module `appirio-tech-react-components` though the latest version available is `1.1.0`.
  I have to keep using the older version as styles from the old version are included in `main.css` in `appirio-tech-react-components` module. If I try to use the latest version for this task, styles from old and new version will interfere and break each other.

  Old version works correctly, but produces the next warnings in the browser console:
  ```
  Warning: Unknown prop `minWidth` on <input> tag. Remove this prop from the element. For details

  warning.js:35 Warning: AutosizeInput: isMounted is deprecated. Instead, make sure to clean up subscriptions and pending requests in componentWillUnmount to prevent memory leaks.

  warning.js:35 Warning: Select: isMounted is deprecated. Instead, make sure to clean up subscriptions and pending requests in componentWillUnmount to prevent memory leaks.
  ```
